app/qml/map/MapCanvas.qml

```diff
@@ -297,6 +272,31 @@ Item {
         }
       }
 
+      // Mouse, stylus and other pointer devices handler
```

Comment:
so the whole fix is just moving this code under `Component`?

Reply:
Yes, but not any `Component`. This component is used in `Loader` with a variable height (see line 204 `height: root.height - root.mapExtentOffset`). The height is a real height of map that is visible and not covered by other elements like form preview, GPS data page or projects panel. This means, that the handler will work only on the part of the map that can be seen.
It is a massive hack, but unfortunately, we still wait for the fix for the upstream bug https://bugreports.qt.io/browse/QTBUG-108821